src/main/java/frc/robot/subsystems/Climb.java

```diff
@@ -25,7 +26,7 @@ public static Climb getInstance() {
 
     CANSparkMax leftMotor, rightMotor;
     public boolean limSwitch;
-    private double gearing = 125;
+    private double gearing = 25;
```

Comment:
as noted previously, if this is a gear ratio and a constant make it `GEAR_RATIO` (i.e. title case) and `private static final double`